portable/ThirdParty/GCC/Posix/port.c

```diff
@@ -280,10 +280,14 @@ struct sigaction sigtick;
 	 * up running on the main thread when it is resumed. */
 	itimer.it_value.tv_sec = 0;
 	itimer.it_value.tv_usec = 0;
+
+	itimer.it_interval.tv_sec = 0;
+	itimer.it_interval.tv_usec = 0;  
 	(void)setitimer( ITIMER_REAL, &itimer, NULL );
 
 	sigtick.sa_flags = 0;
 	sigtick.sa_handler = SIG_IGN;
+	sigemptyset( &sigtick.sa_mask ); 
```

Comment:
Shall we check return values here, or maybe ```(void)``` to mark as we don't care about the return?

(Same comment for the line below, but out of scope of this PR.)